### DISCUSSION TITLE
Ability to create user-created tags for users

### DISCUSSION THREAD

**Discussion by user-727** [Docile-Alligator/Infinity-For-Reddit#972] [Ideas]
Similarly to Boost or RES, I think it would be a useful feature to have the ability to create custom tags and assign them to users. They could appear next to their username or under if not enough space is available, kind of like flairs. 

Also like flairs, tags would be just text (or emojis too I guess) but I think it would be really convenient if they could also have custom colors. So for example, tags created by default would automatically use the "Flair Text Color" as defined in the app theme, but the color could be changed on a per tag basis, giving the users the ability to instantly recognize certain tags without having to read the text, or grouping similar tags together by giving them all the same color.

In my opinion the option to add a tag to a user could be located in the kebab menu on comments (next to bookmark and reply), as well as when you're on a user's profile page. There could also be a page in the settings to manage all the tags you've created and to whom they've been assigned to.

Let me know what you think! Would you have a use for this feature?
Personally it's the one feature I miss the most from other apps and I haven't seen a feature request opened for it yet for Infinity, but definitely let me know if it has already been asked before!

**Comment by QuestioningEspecialy** [comment]
I was actually looking for a way to import my user tags from the discontinued [Slide](https://github.com/Haptic-Apps/Slide), which allows adding a tag from their profile.
I primarily use them as a warning about problematic users (e.g.: bots, rude folk, trolls, etc.). It's been surprisingly helpful every now and then when I stumble on those users in the wild.
Before someone suggests blocking them, that's an extremely limiting solution that prevents me from warning others and being aware of certain things going on in subs I frequent. Some users will benefit from this feature significantly more than others, like mods for example.
